api/views.py

```
@@ -0,0 +1,1021 @@
+    def absolute_stats(self, request: Request) -> Response:
+        """Get absolute statistics without time windowing"""
+        user: User = request.user  # type: ignore
+
+        # Get user domain to calculate patients
+        domain_id = auth_utils.get_domain_id_from_request(request)
+        user_domain = None
+        if domain_id:
+            try:
+                user_domain = UserDomain.objects.get(id=domain_id)
+            except UserDomain.DoesNotExist:
+                pass
```

Ruff (0.8.2)

Use `contextlib.suppress(UserDomain.DoesNotExist)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(UserDomain.DoesNotExist)`

(SIM105)